### PR TITLE
add helpers dir

### PR DESCRIPTION
**Pull request by piyalbasu**

### CODE REVIEW

File: src/helpers/metrics.MD [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+In order to properly transmit your metric to the Amplitude endpoint, you will
+need to provide an Amplitude key. It is best practice to embed this key at build
+time via node environment variable. This key _must_ be called
+`REACT_APP_AMPLITUDE_KEY`. If this key is not passed, in the console, you will
+see a log that says `Not uploading metrics`, along with the metric that
+attempted to transmit.
+
+## Usage
+
+```
```
Comment (marcelosalloum) [inline src/helpers/metrics.MD:16]: We could add the language in the code snippet here:
```suggestion
```ts
```

**Review by marcelosalloum — APPROVED** [review]
LGTM!

File: src/helpers/metrics.MD [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Metrics
```
Comment (quietbits) [inline src/helpers/metrics.MD:1]: Do we need to use uppercase `MD` for the extension?

File: src/helpers/metrics.ts [hunk 3]
```diff
@@ -62,3 +62,7 @@ export const emitMetric = (name: string, body?: any) => {
   });
   uploadMetrics();
```
Comment (quietbits) [inline src/helpers/metrics.ts:63]: In this function, could we also check if Amplitude tracking is not blocked in the browser? Right now, we're getting quite a few error logs from this.

**Review by quietbits — APPROVED** [review]

File: src/helpers/metrics.MD (hunk 1 above)
Comment (piyalbasu) [reply src/helpers/metrics.MD:16]: Good suggestion! I'll add

File: src/helpers/metrics.MD (hunk 2 above)
Comment (piyalbasu) [reply src/helpers/metrics.MD:1]: Oh, we do not - I don't know why I did that lol

**Review by quietbits — APPROVED** [review]

**Pull request merged**